bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java

```diff
@@ -70,7 +73,9 @@
   private volatile ExecutorService batchPool = null;
   private ExecutorService bufferedMutatorExecutorService;
 
-  private BigtableSession session;
+  private final BigtableSession session;
+  // TODO: remove this when introducing BigtableWrapper, once incremental changes are in place
+  private final DataClientWrapper dataClient;
```

Comment:
If I'm not mistaken I think the final goal is to replace BigtableSession with BigtableWrapper?
If that's true, shouldn't we add BigtableWrapper here?  We can add a temporary constructor to it that takes a prebuilt BigtableSession. 

I think we will end up with a lot less noise this way

Reply:
Thanks for the suggestion! I have updated this PR to follow the same approach. PTAL